app/assets/stylesheets/_layout.scss

```diff
@@ -1,5 +1,6 @@
 body {
   background-color: $base-background-color;
+  background-color: #F2F2F2;
```

Comment:
Color literals like `#F2F2F2` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Property `background-color` already defined on line 2<br>Color `#F2F2F2` should be written as `#f2f2f2`